Description: meson 0.58 starts checking for commands that are declared. The
run_all_tests.sh script is nowhere to be found, so just remove that.
Upstream: no
diff --git a/meson.build b/meson.build
index 00f96aa..41a3fe9 100644
--- a/meson.build
+++ b/meson.build
@@ -454,8 +454,6 @@ if check.found()
 endif
 
 
-run_target('test-x', command: [ 'test/run_all_tests.sh' ], depends: [ rofi ])
-
 uncrustify = find_program('uncrustify', required: false)
 if uncrustify.found()
     run_target('indent',
